tests/utils.py

```diff
@@ -14,6 +15,35 @@
 import ga4gh.protocol as protocol
 
 
+class AssertRaises(object):
+    """
+    Decorator that asserts a type of error is raised in the test.
+    Must be placed on a method of a class that derives from
+    unittest.TestCase.
+
+    unittest has an assertRaises method but it doesn't seem to work...
+    """
```

Comment:
It didn't for me.  And even if it did, this class provides a convenient programming model that I don't think `unittest` offers.